tests/test_ensure_database_versions.py

```
@@ -0,0 +1,164 @@
+from unittest.mock import patch, mock_open
+import pytest
+import importlib.util
+import sys
+
+# Import the module under test using importlib
+spec = importlib.util.spec_from_file_location(
+    "ensure_database_versions",
+    Path(__file__).parent.parent / "scripts" / "ensure_database_versions.py",
+)
+if spec is None or spec.loader is None:
+    raise ImportError("Could not load ensure_database_versions module")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)